src/System.Private.Xml/src/System/Xml/Core/XmlReaderSettings.cs

```diff
@@ -753,50 +751,6 @@ internal XmlReader AddConformanceWrapper(XmlReader baseReader)
 
         internal static bool EnableLegacyXmlSettings()
         {
```

Comment:
Is there a good reason for keeping this method around? It is called from just 5 places that are straightforward to fix.